.circleci/config.yml

```diff
@@ -77,6 +77,11 @@ jobs:
           command: |
             wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 20 http://localhost:8080
             :
+      - run:
```

Comment:
this fixes when cypress first 4-5 cypress tests fail because `webpack-server` is not running properly.

Reply:
Just wondering, should this `sleep` be before the `wget` test tho?

Reply:
nope, `wget` tests if server responds, and as you can notice on your local machine `webpack-server` starts responding but its still not ready to be used, and then this sleep fixes that issue 🙂